ESQL test on single value detection fails on Serverless

Test intruduced with https://github.com/elastic/elasticsearch/pull/103150 fails on Serverless


```
    java.lang.AssertionError: Failure at [esql/100_bug_fix:171]: expected [2xx] status code but api [esql.query] returned [405 Method Not Allowed] [{"error":"Incorrect HTTP method for uri [/_query?error_trace=true] and method [POST], allowed: [DELETE, HEAD, GET, PUT]","status":405}]
```

--- a/x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/100_bug_fix.yml
+++ b/x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/100_bug_fix.yml
@@ -1,10 +1,8 @@
 ---
 "Coalesce and to_ip functions":
   - skip:
-      version: all
-      reason: "AwaitsFix https://github.com/elastic/elasticsearch/issues/102871"
-      # version: " - 8.11.99"
-      # reason: "fixes in 8.12 or later"
+      version: " - 8.11.99"
+      reason: "fixes in 8.12 or later"
       features: warnings
   - do:
       bulk:
@@ -129,10 +127,10 @@
 
 
 ---
-"null MappedFieldType on single value detection (https://github.com/elastic/elasticsearch/issues/103141)":
+"null MappedFieldType on single value detection #103141":
   - skip:
-      version: all
-      reason: "AwaitsFix fix https://github.com/elastic/elasticsearch/issues/103561"
+      version: " - 8.12.99"
+      reason: "fixes in 8.13 or later"
   - do:
       indices.create:
         index: npe_single_value_1

